### PR TITLE
Scaffold sound & vision media play

### PR DESCRIPTION
**Pull request by varffs**
this is just a basic framework for the playing functionality. It is quite possible they want to play audio in a custom player not the soundcloud embed (but in the post) and play videos in a lightbox (we will use same one as news)

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -18,6 +18,12 @@
     "lazysizes": "^2.0.4",
     "imagesloaded": "^4.1.1",
     "shufflejs": "^4.0.2",
-    "swiper": "^3.4.0"
+    "swiper": "^3.4.0",
+    "zenscroll": "^3.3.0"
+  },
+  "overrides": {
```
Comment (mixcas) [inline bower.json]: What's `overrides` ?

File: js/main.js [hunk 2]
```diff
@@ -289,6 +290,83 @@ Site.Drawers.About = {
...
+  init: function() {
+    var _this = this;
+
+    _this.bind();
+  },
+
+  bind: function() {
+    var _this = this;
+
+    if ($('.page-sound-and-vision').length) {
```
Comment (mixcas) [inline js/main.js]: Any reason why this condition is here instead of in `init`?

Comment (varffs) [inline js/main.js]: yh. Possibly there will be other media playing pages so figured to init for each page in time

File: bower.json (hunk 1 above)
Comment (varffs) [inline bower.json]: that scroll plugin doesnt have a proper `bower.json` so needs this for the gulp build to work

**Pull request merged**